doc/docs/openapi.md

```diff
@@ -144,18 +144,17 @@ Firstly add dependencies:
 and generate the documentation by importing valid extension methods and explicitly specifying the "3.1.0" version in the OpenAPI model:
 ```scala mdoc:compile-only
 import sttp.apispec.openapi.OpenAPI
-import sttp.apispec.openapi.circe.yaml._ // for `toYaml3_1` extension method
+import sttp.apispec.openapi.circe.yaml._ // for `toYaml` extension method
```

Comment:
I think this whole docs section should be rewritten to show case to generate openapi 3.0.3

Reply:
(as 3.1.0 is now the default)